msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -37,6 +37,491 @@ The following table provides a list of APIs available across the Microsoft Teams
+      "membershipType": undefined,
+      "defaultOneNoteSectionId": undefined,
+      "ownerGroupId": undefined,
+      "ownerTenantId": undefined
+     },
+     "chat": {
+      "id": "19:49683807ffce4318ad6d6d7a24dbde45@thread.tacv2"
+     },
+     "meeting": {
+      "id": "MCMxOTo0OTY4MzgwN2ZmY2U0MzE4YWQ2ZDZkN2EyNGRiZGU0NUB0aHJlYWQudGFjdjIjMTY3ODEwOTM1NDAyMg=="
+     },
+     "sharepoint": undefined,
+     "team": {
+      "internalId": "19:b34aeec3f8e54240a5c283e86bfc4878@thread.tacv2",
+      "displayName": undefined,
+      "type": undefined,
+      "groupId": undefined,
+      "templateId": undefined,
+      "isArchived": undefined,
+      "userRole": 1
+     },
+     "sharePointSite": {
+      "teamSiteUrl": "",
+      "teamSiteDomain": "microsoft.sharepoint.com",
+      "teamSitePath": "",
+      "teamSiteId": "",
+      "mySitePath": undefined,
+      "mySiteDomain": undefined
+     }
+    }
+   ```
+
+   # [Scheduled meeting](#tab/scheduled-meeting)
+
+   The following is a JSON payload response for a scheduled meeting:
+
+   ```json
+     {
+      "app": {
+       "locale": "en-us",
+       "sessionId": "e746c935-0991-47b8-b7f4-16db7646771f",
+       "theme": "default",
+       "iconPositionVertical": 22.998046875,
+       "osLocaleInfo": null,
+       "parentMessageId": "",
+       "userClickTime": 1678023107126,
+       "userFileOpenPreference": "inline",
+       "host": {
+        "name": "Teams",
+        "clientType": "web",
+        "sessionId": "5f2a15f6-75bd-2612-c0ff-2e1085dd165a",
+        "ringId": "general"
+       },
+       "appLaunchId": "e07b6f2c-dc0f-4de2-9b1f-44dba2ea733c"
+      },
+      "page": {
+       "id": "Test",
+       "frameContext": "content",
+       "subPageId": "",
+       "isFullScreen": false,
+       "isMultiWindow": false,
+       "sourceOrigin": NULL
+      },
+      "user": {
+       "id": "e652dd92-dd63-4fcc-b5b2-2005681e8e9f",
+       "displayName": null,
+       "isCallingAllowed": null,
+       "isPSTNCallingAllowed": null,
+       "licenseType": "Unknown",
+       "loginHint": "admin@M365x94626565.onmicrosoft.com",
+       "userPrincipalName": "admin@M365x94626565.onmicrosoft.com",
+       "tenant": {
+        "id": "aa923623-ae61-49ee-b401-81f414b6ad5a",
+        "teamsSku": "enterprise"
+       }
+      },
+      "channel": null,
+      "chat": {
+       "id": "19:meeting_YmU5NWM3NGEtZjMyMi00ZDg4LTk4OGUtMjUzMGJkZjRhMDhm@thread.v2"
+      },
+      "meeting": {
+       "id": "MCMxOTptZWV0aW5nX1ltVTVOV00zTkdFdFpqTXlNaTAwWkRnNExUazRPR1V0TWpVek1HSmtaalJoTURobUB0aHJlYWQudjIjMA=="
+      },
+      "sharepoint": null,
+      "team": null,
+      "sharePointSite": {
+       "teamSiteUrl": "",
+       "teamSiteDomain": "m365x94626565.sharepoint.com",
+       "teamSitePath": "",
+       "teamSiteId": null,
+       "mySitePath": "/personal/admin_m365x94626565_onmicrosoft_com",
+       "mySiteDomain": "m365x94626565-my.sharepoint.com"
+      }
+     }
+
+   ```
+   ---
+
+1. User type
```

Comment:
Please specify that these are obtained for different types of users in the scheduled private meetings

Reply:
Updated that the JSON responses are in scheduled private meetings for each type of user.

![image](https://user-images.githubusercontent.com/93579821/231442379-6d80d334-9156-406b-bbed-a599756e84a5.png)